xtask/src/main.rs

```diff
@@ -14,11 +14,23 @@
 
 //! An xtask binary for managing workspace tasks.
 
+use std::io::Write;
+use std::io::stdin;
+use std::io::stdout;
+use std::path::Path;
 use std::process::Command as StdCommand;
 
 use clap::Parser;
 use clap::Subcommand;
 
+mod colors {
+    pub const RED: &str = "\x1b[31m";
+    pub const GREEN: &str = "\x1b[32m";
+    pub const YELLOW: &str = "\x1b[33m";
+    pub const BLUE: &str = "\x1b[34m";
+    pub const RESET: &str = "\x1b[0m";
+}
```

Comment:
Rust ecosystem has a [`colored`](https://docs.rs/colored/latest/colored/) crate for generating colored  string. You can add it to xtask's dependency and make use of it.